docker-compose.development.yml

```diff
@@ -43,6 +43,7 @@ services:
       - FEATURE_HEARTBEAT_DISABLE=0
       - FEATURE_DB_LISTEN_DISABLE=0
       - CUSTOM_QUICKLINKS=$CUSTOM_QUICKLINKS
+      - GA_TRACKING_ID=none
```

Comment:
what is this used for?

Reply:
This is a Google Analytics tracking id that is forwarded to the frontend, it's not necessary here but it might be good to have all available envs listed here, what do you think?

Reply:
Ah, sounds like a good addition yes. Was just a bit baffled as it is not directly related to the PR in question.